ui/app/src/components/slippagePanel/Index.vue

```diff
@@ -0,0 +1,153 @@
+
```

Comment:
been trying to throw reusable ui components that don't depend on shared state in `shared` which we can later rename `ui`